src/main/java/seedu/tp/group/GroupList.java

```diff
@@ -3,6 +3,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.swing.GroupLayout;
```

Comment:
Is the import used or was it a mistake from intellij autocomplete?

Reply:
Oh, I think this is the latter case. I'll fix that.